docs/content/en/docs/install/integrate.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Integrate KLT
```

Comment:
I don't think this title is descriptive enough. What do you integrate KLT into?
Maybe something like "Use KLT with your Apps"?

Reply:
I changed it to "Integrate KLT with your application" for now.  The prose says that these steps integrate KLT with your cluster, and we are using the term "application" for at least three different things.  Right now, this section basically includes the information  that is in the README file.  It needs more and I'm hoping that I'll figure out the terminology and hence the title as I develop the info more.